README.md

```diff
@@ -1 +1,158 @@
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 HTML/CSS & JavaScript capstone project <a name="about-project"></a>
+
+Portfolio: desktop version  project is the desktop version of my portiflio .
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- linter setup
+- HTML and CSS file creation
+- creating the work and about me sections of my mobile version portiflio
+<p align="right">
+(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo Link](https://solog0039.github.io/capstone-project1.github.io/)
+
+## Presentation of Project
+[Loom Video Link](https://www.loom.com/share/de42a95b30174329a1a534b585346bc2)
+
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+  ## 💻 Getting Started <a name="getting-started"></a>
+
+  To get a local copy up and running it in your machine, follow the next steps.
+
+  ### Prerequisites
+
+  In order to run this project you need:
+
+- Installation of Live Server plugging code editor.
+- A working browser.
+- A code editor.
+- Installation of NPM.
+- Installation of Git.
+  
+### Setup
+
+Copy the projects URL from github then clone from your code editor
+
+### Install
+
+Install almost all dependencies.
+This one will install stylelinter:
+
+> npm install
+
+### Usage
+
+To run this project:
+ 1. open the whole project in VS code 
+ 2. right click on the HTML file
+ 3. select the "open with live server" option.
+ 4. then you can see the on the browser
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Solomon G. Kidanu
+
+- GitHub: [@Solomon](https://github.com/solog0039)
+- Twitter: [@Solomon](https://twitter.com/Solomon57320119)
+- LinkedIn: [Solomon](www.linkedin.com/in/solomon-kidanu-62a994232)
+
+## UI Designer
+
+Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07). But, I personalized for my purposes.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [] HTML file
+- [] CSS file
+- [] Toolbar and headline 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Hi, I would appreciate if you give this project a star(⭐️) this will encourage me to improve it and build the social network of the future.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> "Never stop learning, code never stop teaching"
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] As suggested by the previous reviewer, always. always make sure to give appropriate credit to the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).